instances/molecularEntity/DNQX.jsonld

```diff
@@ -0,0 +1,17 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/molecularEntity/DNQX",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/MolecularEntity",
+  "definition": "DNQX (6,7-dinitroquinoxaline-2,3-dione) is a competitive antagonist at AMPA and kainate receptors, two ionotropic glutamate receptor (iGluR) subfamilies.",
```

Comment:
Please add the source of the definition. Adapt according to the decision made for the naming convention. Do never list synonyms in the definition. We should always just list them under "synonym".

Reply:
I would also delete the further information on the receptors which is irrelevant for this entity. I would though add it's antagonistic competitors for the receptors (and or clearly identify which binding site it uses).